demos/CMakeLists.txt

```diff
@@ -35,3 +35,6 @@ create_demo(Demo_CSG "Demo CSG")
 create_demo(Demo_DoF "Demo DoF")
 create_demo(Demo_VR "Demo Virtual Reality")
 create_demo(Demo_Texturing "Demo Texturing")
+create_demo(Demo_Marble "Demo Marble")

```

Comment:
Please remove Demo Marble and include the code into Demo Texturing in the same way as Demo SCG...